.changeset/cold-singers-suffer.md

```diff
@@ -0,0 +1,5 @@
+---
+"@clerk/clerk-js": minor
+---
+
+Change the value of href of field action achor tags from empty to `#` in order for it to be.
```

Comment:
❓ Something is missing from the end of the sentence? 🤔

Reply:
I've going to completely remove this, because i'm reverting that change.